features/0335-http-over-didcomm/README.md

```diff
@@ -77,16 +81,18 @@ DIDComm messages for this protocol look as follows:
 {
   "@type": "https://didcomm.org/http-over-didcomm/1.0/request",
   "@id": "2a0ec6db-471d-42ed-84ee-f9544db9da4b",
-  "~purpose": [<partial resource uri: scheme://userinfo@host:port/path>],
+  "~purpose": [],
   "method": <method>,
   "resource-uri": <resource uri value>,
   "version": <version>,
   "headers": [],
```

Comment:
http headers should be in key-value, not array.

Reply:
I agree that this would be more natural from some perspectives. However, DIDComm best practice says to use an array of tuples rather than key-value. For an explanation of why, see https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0074-didcomm-best-practices#dont-treat-objects-as-associative-arrays.